.github/workflows/build.yml

```diff
@@ -519,3 +519,60 @@ jobs:
       run: make pythoninfo
     - name: Tests
       run: xvfb-run make buildbottest TESTOPTS="-j4 -uall,-cpu"
+
+  all-required-green:  # This job does nothing and is only used for the branch protection
+    name: ✅ All required checks pass
```

Comment:
Let's remove the emoji, it's a little odd to see a double green (and we could sometimes see a "❌✅ All required checks pass"):

<img width="333" alt="image" src="https://github.com/python/cpython/assets/1324225/7d165a39-b339-4f4c-a177-1ab6ddd2cca2">


```suggestion
    name: All required checks pass
```

Reply:
Done!